pkgs/applications/misc/1password/default.nix

```diff
@@ -32,25 +42,27 @@ stdenv.mkDerivation rec {
   '';
 
   installPhase = ''
-    install -D op $out/bin/op
+    install -D ${mainProgram} $out/bin/${mainProgram}
+    runHook postInstall
   '';
 
-  dontStrip = stdenv.isDarwin;
+  postInstall = "installShellCompletion --cmd ${mainProgram}" + lib.concatMapStrings
+    (s: " --${s} <($out/bin/${mainProgram} completion ${s})") [ "bash" "fish" "zsh" ];
```

Comment:
Please do it like this https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/cluster/argocd/default.nix#L50-L52

Reply:
Looks like I hit the fancy limit :), will do next time I touch this package.

Reply:
Also done in https://github.com/NixOS/nixpkgs/pull/166339.